pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -1928,6 +1928,36 @@ SELECT length(gen_random_uuid()::BYTES), gen_random_uuid() = gen_random_uuid()
 ----
 16 false
 
+query IB
+SELECT length(gen_random_ulid()::BYTES), gen_random_ulid() = gen_random_ulid()
+----
+16 false
+
+query TTTT
+SELECT uuid_to_ulid('0178951c-b665-30a7-19a2-a18999834858'),
+       ulid_to_uuid('01F2AHSDK562KHK8N1H6CR6J2R'),
+       ulid_to_uuid(uuid_to_ulid('0178951c-b665-30a7-19a2-a18999834858')),
+       uuid_to_ulid(ulid_to_uuid('01F2AHSDK562KHK8N1H6CR6J2R'))
+----
+01F2AHSDK562KHK8N1H6CR6J2R  0178951c-b665-30a7-19a2-a18999834858  0178951c-b665-30a7-19a2-a18999834858  01F2AHSDK562KHK8N1H6CR6J2R
+
+let $ulid1
+SELECT gen_random_ulid()
+
+# We need this to ensure that the ULID sorting within the same millisecond works properly
+crdb_internal.pg_sleep(0.001)
```

Comment:
Need to have a `statement ok` above this so the logic test knows to execute the following statement.

By the way are you able to run these logictests locally? To run this one specifically you can do `make testlogic FILES=builtin_function` from the root of the repo.

Reply:
I had no idea TBH. This helped a lot, thanks!